ardrive_network/lib/src/ardrive_network.dart

```diff
@@ -0,0 +1,180 @@
+import 'dart:async';
+import 'dart:collection';
+import 'dart:math';
+
+import 'package:ardrive_network/src/responses.dart';
+import 'package:ardrive_network/src/utils.dart';
+import 'package:dio/dio.dart';
+import 'package:dio_smart_retry/dio_smart_retry.dart';
+import 'package:flutter/foundation.dart';
+import 'package:isolated_worker/js_isolated_worker.dart';
+
+class ArdriveNetwork {
+  int retries;
+  int retryDelayMs;
+  bool noLogs;
+  bool _areScriptsImported = false;
+  int retryAttempts = 0;
+
+  ArdriveNetwork({
+    this.retries = 8,
+    this.retryDelayMs = 200,
+    this.noLogs = false,
+  });
+
+  Dio dio() {
+    final dio = Dio();
+
+    if (!noLogs) {
+      dio.interceptors.add(LogInterceptor());
+    }
+
+    if (retries > 0) {
+      dio.interceptors.add(_getDioRetrySettings(dio));
+    }
+
+    return dio;
+  }
+
+  Future<ArDriveNetworkResponse> get({
+    required String url,
+    bool isJson = false,
+    bool asBytes = false,
+  }) async {
+    checkIsJsonAndAsBytesParams(isJson, asBytes);
+
+    if (kIsWeb) {
+      if (await _loadWebWorkers()) {
+        return await _getWeb(
+          url: url,
+          isJson: isJson,
+          asBytes: asBytes,
+        );
+      }
+    }
+
+    final Map getIOParams = <String, dynamic>{};
+    getIOParams['url'] = url;
+    getIOParams['asBytes'] = asBytes;
+
+    return await compute(_getIO, getIOParams);
+  }
+
+  Future<ArDriveNetworkResponse> getJson(String url) async {
+    return get(url: url, isJson: true);
+  }
+
+  Future<ArDriveNetworkResponse> getAsBytes(String url) async {
+    return get(url: url, asBytes: true);
+  }
+
+  Future<ArDriveNetworkResponse> _getIO(Map params) async {
+    final String url = params['url'];
+    final bool asBytes = params['asBytes'];
+
+    try {
+      Response response;
+      if (asBytes) {
+        response = await dio().get<List<int>>(
+          url,
+          options: Options(responseType: ResponseType.bytes),
+        );
+      } else {
+        response = await dio().get(url);
+      }
+
+      return ArDriveNetworkResponse(
+          data: response.data,
+          statusCode: response.statusCode,
+          statusMessage: response.statusMessage,
+          retryAttempts: retryAttempts);
```

Comment:
Add a ` , ` to a better format

```suggestion
          retryAttempts: retryAttempts,);
```

Reply:
Fixed here [`1e2c2df` (#788)](https://github.com/ardriveapp/ardrive-web/pull/788/commits/1e2c2dfa418686a364dab3849f9c3cabb5764fee)